src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -73,6 +73,13 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
   @Output()
   public activeIndexChange = new EventEmitter<number>();
 
+  /**
+   * Fires when the order of repeater items change.
```

Comment:
Because it's the "order" that changes, not the "repeater items," the last word here should be "changes," not "change."

That said, can this be done programmatically? If not (i.e., if it can only be done by users), then I'd change this to "Fires when users change the order of repeater items." This avoids the awkwardness of the grammatically correct verb sounding wrong. But if it can be done programmatically, I think we need top stick with the original version.

Reply:
Yeah, I like this suggestion - it makes it closer to the actual intent: when users do something.
Changed 👍